javascript/ql/src/semmle/javascript/StandardLibrary.qll

```diff
@@ -209,6 +209,26 @@ private class PromiseFlowStep extends DataFlow::AdditionalFlowStep {
   }
 }
 
+/**
+ * A data flow edge from the exceptional return of the promise executor to the promise catch handler.
+ * This only adds an edge from the exceptional return of the promise Executor and to a `.catch()` handler.
+ * Missing are (at least): 
```

Comment:
This sounds like something that should be put into an issue, not a code comment.